### ISSUE TITLE
Map zooming and markers

### ISSUE THREAD

**Issue by jan-christiansen** [Outdooractive/route-me#48]
When performing a double tap on a marker, the map zooms in. In contrast, when performing a two finger double tap where one of the fingers hits a marker, does not cause a zoom out but a zoom in. Furthermore, pan gestures do not zoom the map if one of the
fingers hits a marker. Would it be possible to pass all gestures except for taps on markers to the map. If there are lots of markers on the map, it is quite difficult to zoom the map by a pan gesture.

**Comment by trasch** [comment]
I know what you mean... I'll have a look at this.

**Comment by jan-christiansen** [comment]
Have you made any progress on this? If not, I would like to check whether I can change the behavior myself.

**Comment by trasch** [comment]
No, I am sorry but I simply didn't have the time yet. If you find something it would be nice if you could open a pull request so that I can take it into the project, but I won't have time myself until later in may.

**Comment by jan-christiansen** [comment]
I have changed the implementation of gesture recognizers in a branch called `gesture-recognition`. All recognizers except for the pan gesture recognizer are added to the map view. The pan gesture recognizer that is used to drag annotations is added to the scroll view. To pass pans that did not hit a draggable annotation to the scroll view, I have implemented `gestureRecognizerShouldBegin:`. I did not check whether the implementation is correct if you use draggable annotations as I do not use these annotations, so there might still be some bugs. All gestures that are not handled by the scroll view are handled by the map view.
